integrationTests/json/scenariosTestCommon.go

```diff
@@ -50,12 +52,16 @@ type ScenariosTestBuilder struct {
 // ScenariosTest will create a new ScenariosTestBuilder instance
 func ScenariosTest(t *testing.T) *ScenariosTestBuilder {
 	return &ScenariosTestBuilder{
-		t:                   t,
-		folder:              "",
-		singleFile:          "",
-		executorLogger:      nil,
-		executorFactory:     nil,
-		enableEpochsHandler: worldmock.EnableEpochsHandlerStubAllFlags(),
+		t:               t,
+		folder:          "",
+		singleFile:      "",
+		executorLogger:  nil,
+		executorFactory: nil,
+		enableEpochsHandler: &worldmock.EnableEpochsHandlerStub{
+			IsFlagEnabledCalled: func(flag core.EnableEpochFlag) bool {
+				return flag != vmhost.ValidationOnGobDecodeFlag // relevant for big float tests. TODO: remove this when completely migrate to go1.22
```

Comment:
this is not good. Shouldn't be default for all the tests. So will rewrite it tomorrow this part.